LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationNetworkServerStartup.cs

```diff
@@ -40,7 +41,10 @@ public void ConfigureServices(IServiceCollection services)
             _ = services.AddLogging(loggingBuilder =>
                         {
                             _ = loggingBuilder.ClearProviders();
-                            var logLevel = (LogLevel)int.Parse(NetworkServerConfiguration.LogLevel, CultureInfo.InvariantCulture);
+                            var logLevel = int.TryParse(NetworkServerConfiguration.LogLevel, out var logLevelNum)
```

Comment:
Shoudl we use the overload TryParse (string? s, System.Globalization.NumberStyles style, IFormatProvider? provider, out int result); to be stull using invariant culture?

Reply:
Done in b3c7f3921621aa11d8efa6036a95f8e43788fee6